e2e/cypress/integration/api/planetReviewsRouter.spec.js

```diff
@@ -0,0 +1,81 @@
+/// <reference types="Cypress" />
+
+context("As a user sending HTTP requests to /api/v1/planetReviewsRouter", () => {
+
+  describe("If I send a POST request to /planets/:planetId/reviews", () => {
```

Comment:
its not a big deal, but ideally we avoid "I" in this context. The user? the developer? 

in fact, we could swap these up entirely, because its not a user that is triggering these per se 

```
context("POST  /api/v1/planets/:id/reviews" ...
     describe("when request body is valid"....
```